test/e2e/specs/sidebar-behaviour.test.js

```diff
@@ -0,0 +1,80 @@
+/**
```

Comment:
`-behaviour` is kinda redundant as a suffix, and doesn't add much value. Any test could be called as such, `undo-behavior.test.js`, `writing-flow-behavior.test.js`, etc.

Also, typo since it's spelled "behavior", not "behaviour" :trollface: 🇺🇸